xbmc/network/CMakeLists.txt

```diff
@@ -44,6 +44,16 @@ if(SHAIRPLAY_FOUND)
                       DllLibShairplay.h)
 endif()
 
+include_directories(${CORE_SOURCE_DIR}/lib/libUPnP
```

Comment:
please use this instead (needs to go after `core_add_library`). That way the include dirs are not propagated into the subdirectories and the compiler uses them just where they are needed.
```
target_include_directories(network PRIVATE $ )
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Platinum/Source/Core
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Platinum/Source/Platinum
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Platinum/Source/Devices/MediaConnect
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Platinum/Source/Devices/MediaRenderer
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Platinum/Source/Devices/MediaServer
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Platinum/Source/Extras
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Neptune/Source/System/Posix
                           ${CORE_SOURCE_DIR}/lib/libUPnP/Neptune/Source/Core)
```